app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -51,6 +51,7 @@ def new
       end
 
       flash_error(result.errors) unless result.success?
+      increment_mfa_selection_attempt_count(:webauthn)
```

Comment:
I don't think this is accurately counting attempts. If I sat on the WebAuthn setup page and refreshed the page, it would count that as an attempt, but that's not really an attempt. I think we'd want this inside the `result.errors.present?` check above.

Reply:
I moved it there. I don't honestly know why that works. Do we expect errors to be present every time?